HydroModelBuilder/ModelInterface/flopyInterface/viz/MT3D_PP_viz.py

```diff
@@ -256,3 +267,155 @@ def viewConcsByZone(self, nper='all', specimen=None):
 
     plt.show()
 # End viewConcsByZone()
+
+
+def _plot_obs_vs_sim(self, obs_set, obs_sim_zone_all, unc=None):
+    scatterx = np.array([h[0] for h in obs_sim_zone_all])
```

Comment:
docstring?

Reply:
@ConnectedSystems , Ok I have removed duplicate functions and created a new "Utilities" module called model_assessment.py which contains these repetitive functions and includes doc strings